astropy/table/meta.py

```diff
@@ -76,7 +76,7 @@ def _construct_odict(load, node):
       >>> yaml.load('''!!omap [ foo: bar, mumble: quux, baz : gorp ]''')  # doctest: +SKIP
       OrderedDict([('foo', 'bar'), ('mumble', 'quux'), ('baz', 'gorp')])
     """
-    omap = OrderedDict()
+    omap = {}
```

Comment:
I think the docstring needs to be adjusted here! Note that the doctest is skipped - I think that may be from a time when `yaml` was not a required dependency.

Reply:
I ended up removing the example entirely. It turns out that it didn't work as shown. First there were no imports so there was ambiguity about the `yaml` symbol (from astropy or pyyaml?). But more importantly it didn't show that this private function needs to be registered as a constructor in a custom Loader subclass. The comment now just refers to the astropy function where it is used and that should be fine.